### ISSUE TITLE
Launch configuration should have variables and prompts for parameters

### ISSUE THREAD

**Issue by cristalp** [de-jcup/eclipse-bash-editor#267]
In other launch configurations (Java etc.), it's possible to add variables, like ${current_date} or ${file_prompt}. This seems to work for Environment variables, but not for the parameters. This is a pity, because it's a very powerful feature in Eclipse.

I don't know Eclipse from a plugin developers perspective, but if this is easy to integrate, it would be a great feature!

Thanks for your great plugin, I really love it!

**Comment by de-jcup** [comment]
You are right: At the environment tab you are able to add "variables" and it seems to work... But at runtime the variables are currently not evaluated.

To understand it correctly:
- you would like to have the possibilty to add something like "$(current_date)" as a parameter (like in next picture)
- you would like to have those parameters replaced at runtime
- out-of-scope: code completion while entering parameters

![image](https://user-images.githubusercontent.com/11644753/227866575-ff64a799-5ed7-439f-b1c4-1ecc09693535.png)

When I start currently the launch configuration following is shown:
![image](https://user-images.githubusercontent.com/11644753/227866923-33dc9085-b485-4f0d-8ccf-cded79e4ec9b.png)

In my example you want to have the 4th parameter to be the current date?

**Comment by cvgaviao** [comment]
I also have tried to use Eclipse Variables today and it didn't work. It is not being interpolated once the debug is started.
In my case I just used the `project_loc` one in order to get the project's root directory: `${project_loc:/my-project}` so I can use it inside the script.

Btw. this is a very nice plugin and it's debug support saved my life today :)

**Comment by cristalp** [comment]
Sorry for not answering, it looks like the GitHub mail got caught in the spam folder...

Yes, you unterstand the first two bullet points correctly. Since I don't know whether this is a standard functionality of Eclipse or just part of the Java launch configuration framework, I don't know whether you have to implement all of these features yourself.

But nonetheless, it would be very cool to have parameters for file prompts etc.    :-)
